### DISCUSSION TITLE
wxSqlite3-3.4.9.6

### DISCUSSION THREAD

**Discussion by Carluccio** [utelle/wxsqlite3#111] [General]
Good morning, I work with windows 10 and compile with mingw-64 and tdm-64.
wxsqlite3-3.4.9.6, 3.4.9.5 and 3.4.9.4 does not compile:
"==== Building wxsqlite3 (release_dll_win64) ===="
wxsqlite3.cpp
Linking wxsqlite3
D:/TDM-GCC-64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld. exe: obj/gcc/Win64/Release_DLL/wxsqlite3/sqlite3mc_amalgamation.o:sqlite3mc_amalgamation.c:(.text+0x5cf14): undefined reference to `strtoflt128'
D:/TDM-GCC-64/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld. exe: obj/gcc/Win64/Release_DLL/wxsqlite3/sqlite3mc_amalgamation.o:sqlite3mc_amalgamation.c:(.text+0x5cf2f): undefined reference to `modfq'
collect2.exe: error: ld returned 1 exit status
wxsqlite3.make:195: recipe for target '../lib/gcc_x64_dll/libwxsqlite3_msw32u.dll' failed
mingw32-make[1]: *** [../lib/gcc_x64_dll/libwxsqlite3_msw32u.dll] Error 1
Makefile:85: recipe for target 'wxsqlite3' failed
mingw32-make: *** [wxsqlite3] Error 2

**Comment by utelle** [comment]
Unfortunately, I have to confirm this bug. Thanks for reporting.

The VSV extension enables the `long double` (aka `__float128`) type, if the compiler is GCC and the platform is WIN64.

There are 2 possible workarounds:

1. Disable extension VSV (if you don't need it) - remove all occurences of `-DSQLITE_ENABLE_VSV=1` in the makefiles
 or
3. Add `libquadmath` to the list of link libraries - add `-lquadmath` after `-luxtheme` in the makefiles

**Comment by utelle** [comment]
I adjusted the makefiles to fix the problem (see issue #112).
